Whisky/Utils/Logger.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  Logger.swift
+//  Whisky
+//
+//  Created by Isaac Marovitz on 08/06/2023.
+//
+
+import Foundation
+import OSLog
+
+class Log {
+    static let logsFolder = FileManager.default.urls(for: .libraryDirectory,
+                                                    in: .userDomainMask)[0]
+        .appendingPathComponent("Logs")
+        .appendingPathComponent("Whisky")
```

Comment:
`URL` has loads of [built-in shortcuts](https://developer.apple.com/documentation/foundation/url/3988469-librarydirectory) now that avoid the need to use `urls...[0]`
```suggestion
    static let logsFolder = URL.libraryDirectory
        .appendingPathComponent("Logs")
        .appendingPathComponent("Whisky")
```